lib/dce_lti/controller_methods.rb

```diff
@@ -1,9 +1,7 @@
 module DceLti
   module ControllerMethods
     def authenticate_via_lti
-      if ! current_user
```

Comment:
I know this will seem anal, but I generally only use the `function unless foo` construction for guard clauses, and I don't really think of this as a guard clause. I could be convinced otherwise.